ext/io/winpipe.rs

```diff
@@ -43,24 +42,21 @@ pub fn create_named_pipe() -> io::Result<(RawHandle, RawHandle)> {
 fn create_named_pipe_inner() -> io::Result<(RawHandle, RawHandle)> {
   static NEXT_ID: AtomicU32 = AtomicU32::new(0);
   // Create an extremely-likely-unique pipe name from randomness, identity and a serial counter.
-  let pipe_name = format!(
-    r#"\\.\pipe\deno_pipe_{:x}.{:x}.{:x}\0"#,
+  let pipe_name_utf8 = format!(
+    r#"\\.\pipe\deno_pipe_{:x}.{:x}.{:x}"#,
```

Comment:
Do we handle name collisions?

Reply:
Via retry, unfortunately the only way to do this on Windows